server/rest/src/test/java/org/infinispan/rest/resources/AbstractRestResourceTest.java

```diff
@@ -89,7 +89,7 @@ protected GlobalConfigurationBuilder getGlobalConfigForNode(int id) {
    protected void addSecurity(GlobalConfigurationBuilder globalBuilder) {
       globalBuilder.security().authorization().enable().principalRoleMapper(new IdentityRoleMapper())
                    .role("ADMIN").permission(AuthorizationPermission.ALL)
-                   .role("USER");
+                   .role("USER").permission(AuthorizationPermission.ALL);
```

Comment:
Same as above, why does USER need to have the same permissions as ADMIN?

Reply:
task execution as far as I remember

Reply:
Maybe add only EXEC?

Reply:
ok

Reply:
in any case adding permissions is mandatory here

Reply:
the xml validation does not work if we don't. the permissions property is mandatory

Reply:
So the builder should validate it, or does it assume some default ones? /cc @tristantarrant

Reply:
I get on validation that the permissions are mandatory and this is a required field in the xsd
Maybe the java code should be improved on that?
In any case, I added READ WRITE and EXEC. But for the cache manipulation on protobuf, I need to keep on the subject USER the role that it's needed for that.
I added a commit so you can check the changes easily. I rather not spent more time on this right now, since this should be backported asap